### DISCUSSION TITLE
Remove block-renderer and layouts from the components folder

### DISCUSSION THREAD

**Discussion by pigeonfresh** [mubanjs/muban-skeleton#20] [Ideas]
Right now the components folder hosts layouts and block-renderer templates and logic. To me it would make more sense to have  these in the enclosing folder `src` and keep the components folder for cms-blocks, atoms or whatever lingo we are using for our website building blocks. I know @ThijsTyZ already set this up for (some) projects.

Thoughts?

**Comment by ThijsTyZ** [comment]
Agree
